src/runge_kutta.jl

```diff
@@ -0,0 +1,473 @@
+# Explicit Runge-Kutta solvers
+##############################
+# (Hairer & Wanner 1992 p.134, p.165-169)
+
+###########################################
+# Tableaus for explicit Runge-Kutta methods
+###########################################
+
+immutable TableauRKExplicit{Name, S, T} <: Tableau{Name, S, T}
```

Comment:
I get `ERROR: type: instantiate_type: expected TypeConstructor, got Type{(Any...,)}` on this line with Julia 0.3.8-pre+2.